go-controller/pkg/ovn/ovn.go

```diff
@@ -130,7 +172,7 @@ type Controller struct {
 	defaultGatewayCOPPUUID string
 
 	// Controller used for programming OVN for egress IP
-	eIPC egressIPController
+	eIPC *egressIPController
```

Comment:
nit: is this an optimization, or is it actually required for this PR ?

Reply:
just an optimization.